lldb/test/API/python_api/type/main.cpp

```diff
@@ -27,6 +27,7 @@ class Task {
     enum E : unsigned char {} e;
     union U {
     } u;
+    static constexpr long static_constexpr_field = 47;
```

Comment:
Should we have an XFAIL test for the non-constant case?

Reply:
Good idea.